### ISSUE TITLE
Index.html not compiling properly

### ISSUE THREAD

**Issue by TheBull963** [rhazdon/hugo-theme-hello-friend-ng#26]
When I run hugo to compile the site for use the Index.html file does not compile properly.

This section:

```
<main aria-role="main">
        <div>
            <h1><a href="/posts"> {{ .Site.Title }} </a></h1>

            {{- with .Site.Params.homeSubtitle }}
                <p>{{.}}</p>
            {{- end }}

            {{- with .Site.Params.social }}
                <div>
                    {{ partial "social-icons.html" . }}
                </div>
            {{- end }}
        </div>
    </main>
```

Turns into:

  ```
 <main class="posts">
        <h1></h1>
    </main>

```

**Comment by Ezequieltbh** [comment]
Same issue here!
Any advice in order to fix that?

**Comment by Ezequieltbh** [comment]
I tried with the "exampleSite"  provided in the root folder of this project and with the last version of this theme.
Still not working...

**Comment by rhazdon** [comment]
Is it related to this issue here? https://github.com/rhazdon/hugo-theme-hello-friend-ng/issues/14
Sounds like that...

**Comment by Ezequieltbh** [comment]
@rhazdon You are right!
That is the bug.
Thanks for the response, and great work with the theme, it's awesome!

**Comment by ChetanBhasin** [comment]
Does anyone have any idea what's causing this problem? Anything we can do to help?

**Comment by Reijnn** [comment]
Same issue still exists. It seems to recognize the page as a sub page.
The title of the tab changes to :: <.Site.Title> as if its not on the home page.

**Comment by siyagule** [comment]
Issue resolved  [#14](https://github.com/rhazdon/hugo-theme-hello-friend-ng/issues/14) here
You need to comment out the taxonomies section
